idp_task1.cpp

```diff
@@ -1,55 +1,61 @@
 #include "socket_wrapper.hpp"
-#include "util.h"
+#include "socket_exception.hpp"
+#include <iostream>
+#include <string.h>
+#include <arpa/inet.h>
```

Comment:
Is it really needed here?

Reply:
yes, if we use inet_ntoa, strlen, strcpy in this file